recipes/openblas/all/conanfile.py

```diff
@@ -68,7 +68,7 @@ def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
 
-    def validate(self):
+    def validate_build(self):
```

Comment:
I'd say it is likely that the only condition that needs to be moved to ``validate_build()`` is the one:

```python
raise ConanInvalidConfiguration("Cross-building not implemented")
```
All the other condition seem good as ``validate()``, those are not about cross-building but they are also meant for consumption of the package, irrespective of the build, so those are good in ``validate()``